dashboard/app/dsl/script_dsl.rb

```diff
@@ -81,8 +81,9 @@ def level_concept_difficulty(json)
 
   string :prompt
 
-  def assessment(name)
-    level(name, {assessment: true})
+  def assessment(name, properties = {})
+    properties[:assessment] = true
+    level(name, properties)
```

Comment:
`assessment <name>` is used a lot, and `assessment <name>, active: false` will be used if Baker goes ahead with the assessment swap they were asking about last week.

I guess they could just switch to `level <name>, assessment: true, active: false`, but this seems nicer.